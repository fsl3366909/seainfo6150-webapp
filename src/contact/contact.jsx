import React from 'react'

import { Switch, Route, Link } from "react-router-dom";

const Home = () => {
    return (
        <div>
            <h1>Contact Us</h1>
            <br />
            <br />
            <br />
            <br />
            <p>Name: Shaolong Feng</p>
            <br />
            <br />
            <p>Email: feng.shao@northeastern.edu</p>
        </div>
    )
}

export default Home
