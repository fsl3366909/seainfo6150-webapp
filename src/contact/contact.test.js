import React from "react";
import contact from "./contact";


it("renders the home component correctly", () => {
    const { container } = render(<contact />);
    expect(container).toMatchSnapshot();
});
