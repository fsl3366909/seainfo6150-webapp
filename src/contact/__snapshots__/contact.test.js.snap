// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <contact />
</div>
`;
